docs/Config_Reference.md

```diff
@@ -318,6 +317,83 @@ radius:
 #   just prior to starting a probe operation. The default is 5.
 ```
 
+### Deltesian Kinematics
+
+See [example-deltesian.cfg](../config/example-deltesian.cfg) for an
+example linear delta kinematics config file. See the
```

Comment:
Should be "for an example deltesian kinematics file"?

-Kevin

Reply:
Oups. Thank you for your vigilance. It is now corrected.